### PR TITLE
Run phase index results

### PR DESCRIPTION
**Pull request by jtaleric**
Previously, the run phase would simply run the workload but do nothing
with the results. Now we store the results into ES, if provided.

### CODE REVIEW

File: attack_run.sh [hunk 1]
```diff
@@ -3,9 +3,18 @@
...
 export QUAY_URL=<URL>
+export UUID=$(uuidgen)
+export DURATION=120
 export PARALLELISM=10
-export NUM_USERS=1000
-export RATE=10
+export NUM_USERS=10
+export RATE=40
+export PREFIX=perf-test
+export ES=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
```
Comment (kurtismullins) [inline attack_run.sh:12]: Do you want this in your public Github repo?

File: attack_run.sh [hunk 2]
```diff
@@ -3,9 +3,18 @@
...
+export ES_PORT=80
+export DB=mysql57
+export TEST_NAME=rook_test
+export QUAY_VERSION=3
```
Comment (kurtismullins) [inline attack_run.sh:16]: (Nitpick) Want to change the default here to `"3.3.0"` or something along those lines? Soon, we'll have `3.3.1` and the next major release will be `3.4.0`

File: run-script.sh [hunk 3]
```diff
@@ -1,59 +1,92 @@
...
 prefix=perf-test
+uuid=${UUID}
 num_users=${NUM_USERS:-30}
 rate=${RATE:-10}
 quay=${QUAY_URL}
 duration=${DURATION:-10}
 pick=$((1 + RANDOM % ${num_users}))
 
+prefix=${PREFIX:-perf-test}
+elastic=${ES:-search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com}
```
Comment (kurtismullins) [inline run-script.sh:15]: (Same as above)
Do you want to keep this in your public Github repo?

File: attack_run.sh (hunk 1 above)
Comment (jtaleric) [reply attack_run.sh:12]: yeah. It is everywhere. No problem!

File: run-script.sh (hunk 3 above)
Comment (jtaleric) [reply run-script.sh:15]: :) yeah, i don't care about this machine.

File: attack_run.sh (hunk 2 above)
Comment (jtaleric) [reply attack_run.sh:16]: Patch coming.

**Pull request merged**